gulpfile.js

```diff
@@ -391,30 +413,22 @@ function release_win(arch) {
 
 // Create distribution package (zip) for windows and linux platforms
 function release(arch) {
-    var src = path.join(appsDir, pkg.name, arch);
-    var output = fs.createWriteStream(path.join(releaseDir, get_release_filename(arch, 'zip')));
-    var archive = archiver('zip', {
-        zlib: { level: 9 }
-    });
-    archive.on('warning', function (err) { throw err; });
-    archive.on('error', function (err) { throw err; });
-    archive.pipe(output);
-    archive.directory(src, 'Betaflight Configurator');
-    return archive.finalize();
+    var src = path.join(appsDir, pkg.name, arch + '/**');
```

Comment:
If we are using the `path.join()` to join the parts, I think is better to use it for all the parts and not use the `+` in the latest parameter:
`var src = path.join(appsDir, pkg.name, arch, '**');`